src/api/public/apidocs-new/components/schemas/person_tokens.yaml

```diff
@@ -21,6 +21,10 @@ properties:
           type: string
           xml:
             attribute: true
+        triggered_at:
+          type: string
```

Comment:
I think we're missing the `format: date-time` here. See the [OpenAPI specification for data types](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#data-types).

Reply:
I thought we are not yet dealing with the formats in openapi docs, but fine for me, added the format :)